### PR TITLE
Added placeholder and buttonLabels properties for FilePicker

### PR DESCRIPTION
**Pull request by narekmal**
Addresses #531

### CODE REVIEW

File: src/file-picker/src/FilePicker.js [hunk 1]
```diff
@@ -47,6 +47,20 @@ export default class FilePicker extends PureComponent {
...
 
+    /**
+     * The placeholder attribute of the input.
+     */
+    placeholder: PropTypes.string,
+
+    /**
+     * Button labels in different states of the component.
+     */
+    buttonLabels: PropTypes.shape({
```
Comment (mshwery) [inline src/file-picker/src/FilePicker.js]: I think we could make all of these top level props instead and simplify some of the logic below.

**Comment by mshwery** [comment]
Could you add a description to this PR describing the change and why?

**Comment by narekmal** [comment]
PR addresses issue #531  by adding properties to FilePicker component for overriding default input placeholder, select file button label, replace file button label, replace files button label

File: src/file-picker/src/FilePicker.js [hunk 2]
```diff
@@ -108,21 +116,19 @@ export default class FilePicker extends PureComponent {
     let buttonText
     if (files.length === 0) {
       buttonText =
-        buttonLabels === undefined || buttonLabels.selectLabel === undefined
+        selectFileButtonLabel === undefined
           ? 'Select file'
```
Comment (mshwery) [inline src/file-picker/src/FilePicker.js]: If you use defaultProps you shouldn't need to do these `undefined` checks which will clean up the code quite a bit.

**Pull request closed without merging**